### PR TITLE
Renaming extrinsic/intrinsic coordinates of hyperbolic spaces  in hyperbolic.py

### PR DESCRIPTION
**Pull request by soma2000-lang**
#1246 

## Checklist

- [x] My pull request has a clear and explanatory title.
- [x] If neccessary, my code is [vectorized](https://www.geeksforgeeks.org/vectorization-in-python/).
- [x] I have added appropriate unit tests.
- [x] I have made sure the code passes all unit tests. (refer to comment below)
- [x] My PR follows [PEP8](https://peps.python.org/pep-0008/) guidelines. (refer to comment below)
- [x] My PR follows [geomstats coding style](https://github.com/geomstats/geomstats/blob/master/docs/contributing.rst#coding-style-guidelines) and API.
- [x] My code is properly documented and I made sure the documentation renders properly. ([Link](https://github.com/geomstats/geomstats/blob/master/docs/contributing.rst#documentation))
- [x] I have linked to issues and PRs that are relevant to this PR.

### CODE REVIEW

File: geomstats/geometry/_hyperbolic.py [hunk 1]
```diff
@@ -46,71 +46,71 @@ def _extrinsic_to_extrinsic_coordinates(point):
...
         Returns
         -------
-        point_extrinsic : array-like, shape=[..., dim + 1]
-            Point in hyperbolic space in extrinsic coordinates.
+        point_extrinsic-hyperboloid: array-like, shape=[..., dim + 1]
+            Point in hyperbolic space in extrinsic-hyperboloidcoordinates.
         """
         coord_0 = gs.sqrt(1.0 + gs.sum(point**2, axis=-1))
-        point_extrinsic = gs.concatenate([coord_0[..., None], point], axis=-1)
+        point_extrinsic-hyperboloid= gs.concatenate([coord_0[..., None], point], axis=-1)
```
Comment (lpereira95) [inline geomstats/geometry/_hyperbolic.py:77]: Thanks for the contribution @soma2000-lang.

Notice that `point_extrinsic-hyperboloid` is not a valid python name (that's the reason for `_`).

(This happens in other places in the code)

**Comment by lpereira95** [comment]
@soma2000-lang are you considering to continue this? If not, I'll close to reduce the clutter here.

**Comment by soma2000-lang** [comment]
@yes i will work on it

**Comment by soma2000-lang** [comment]
@lpereira95  I have made the changes please review.

**Comment by lpereira95** [comment]
@soma2000-lang, there's still places in the code that do not have proper syntax. The tests should pass for the changes to be valid.

**Comment by soma2000-lang** [comment]
Ok

**Pull request closed without merging**